changelog.d/20250813_113819_PL-133945_nginx-append-to-commonhttpconfig_scriv.md

```diff
@@ -0,0 +1,21 @@
+<!--
+
+A new changelog entry.
+
+Delete placeholder items that do not apply. Empty sections will be removed
+automatically during release.
+
+Leave the XX.XX as is: this is a placeholder and will be automatically filled
+correctly during the release and helps when backporting over multiple platform
+branches.
+
+-->
+
+### Impact
+
+- The Nginx worker processes will be restarted while applying this change due to changes to it's configuration. Since nginx can handle this using a hot reload, we expect no effective down-time.
+
+
+### NixOS XX.XX platform
+
+- Adjust the nginx configuration by moving general configuration like log formats up above the server configuration blocks (PL-133945)
```

Comment:
> so that for example log formats can be reused in the virtual hosts.

I suggest that you mention this improvement in the changelog as well.

Suggestion:
```suggestion
- nginx: platform-defined general configuration directives like log formats have been moved up above the server configuration blocks, allowing their re-use in virtual host config (PL-133945)
```